tests/integration/routes/test_error_page_templates.py

```diff
@@ -0,0 +1,233 @@
+from mock import Mock, patch
```

Comment:
We should have tests for the content which covers, the scenarios:
- authenticated
- not authenticated, but cookie exists
- not authenticated and no cookie. For this both the business and other link needs to be checked.

Reply:
I've added what I could and tried more descriptive test names but certain errors are impossible to recreate without being authenticated (eg. 500, submission failed) and certain occur only when unauthenticated (401)?

Reply:
Apart from `submission failed`, the rest can be under those 3 categories.